DigitalLearningSolutions.Web/Views/TrackingSystem/Shared/_NavMenuItems.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@using DigitalLearningSolutions.Web.Helpers
+@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+
+@if (User.HasClaim(c => c.Type == CustomClaimTypes.UserUserAdmin)) {
+  if (User.HasClaim(c => c.Type == CustomClaimTypes.UserCentreAdmin || c.Type == CustomClaimTypes.UserCentreManager)) {

```

Comment:
Could we add this to CustomClaimHelper as "HasCentreAdminPermissions" (as UserCentreManager should have same permissions as UserCentreAdmin and it will be much easier to read: User.HasCentreAdminPermissions)?

We should probably also modify the Policy we're using on Authorization of the existing TrackingSystem pages (can you add that on the JIRA ticket 🙏 )?

Reply:
So the policy changes from checking if they have a UserCentreAdmin claim to if they have this new bool based on checking both the UserCentreAdmin and UserCentreManage claim.

Reply:
Nice - could we actually add the super admin to the list too as they're implicitly centre admins (UserUserAdmin)?